packages/inventory/src/Stocktake/DetailView/SidePanel.tsx

```diff
@@ -21,13 +23,19 @@ import {
 const AdditionalInfoSection: FC = () => {
   const t = useTranslation('common');
 
-  const { comment, update } = useStocktakeFields('comment');
+  const { comment, user, update } = useStocktakeFields(['comment', 'user']);
   const [bufferedComment, setBufferedComment] = useBufferState(comment);
   const isDisabled = useIsStocktakeDisabled();
 
   return (
     <DetailPanelSection title={t('heading.additional-info')}>
       <Grid container gap={0.5} key="additional-info">
+        <PanelRow>
+          <PanelLabel>{t('label.entered-by')}</PanelLabel>
+          <PanelField>{user?.username}</PanelField>
+          {user?.email ? <InfoTooltipIcon title={user.email} /> : null}
```

Comment:
username is mandatory, but the email isn't - just not showing the info icon 🤷

Reply:
still hoping to get more user details.. enough for now though